### ISSUE TITLE
Improvements to installation process

### ISSUE THREAD

**Issue by Diggsey** [rust-lang-nursery/rustup.rs#386]
I feel that the initial installation screen could be more friendly. These are some things that I'd like to change, but many are subjective. For reference, here is the current text:
```

Welcome to Rust!

This will download and install the official compiler for the Rust
programming language, and its package manager, Cargo.

It will add the `cargo`, `rustc`, `rustup` and other commands to
Cargo's bin directory, located at:

C:\Users\diggs\.cargo\bin

This path will then be added to your `PATH` environment variable by
modifying the HKEY_CURRENT_USER/Environment/PATH registry key.

You can uninstall at any time with `rustup self uninstall` and
these changes will be reverted.

WARNING: This is beta software.

To cancel installation, type "n", or for more options type "a",
then press the Enter key to continue.

Press the Enter key to install Rust.
a

Selected installation options:

     default toolchain: stable
  modify PATH variable: yes

I'm going to ask you the value of each these installation options.
You may simply press the Enter key to accept the default.

...
```

1) Indentation
The path and possibly the paragraphs could be indented a few spaces to give some structure to the message.

2) Colour/formatting
The heading could be more prominant. References to things currently delineated by backticks could be formatted differently.

3) Installation settings
The default installation settings are only shown *after* the user has made the choice to customize them. It would be more useful to show the settings first, and then ask the user whether to accept or customize them.

4) Options
Present the options as a list instead, eg.
```
The current installation options are XYZ

1) Proceed with installation (default)
2) Customize installation
3) Cancel installation

Enter choice (1-3):
```

**Comment by brson** [comment]
> The path and possibly the paragraphs could be indented a few spaces to give some structure to the message.

I agree on the paths at least. Indent like a markdown code block.



> The heading could be more prominant. References to things currently delineated by backticks could be formatted differently.

Yes! I definitely would like to see some basic console formatting to bring out important info.



> Present the options as a list instead, eg.

I'm happy to try this.

**Comment by Diggsey** [comment]
These are implemented in #418
